docs/docs/licenses/cancel-many.md

```diff
@@ -0,0 +1,35 @@
+---
+sidebar_position: 1
+---
+
+# Cancel many Licenses
+
+This method will cancel many adhoc Licenses
+
+## Code Sample
+
+```typescript
+const { Salable } = require('@salable/node-sdk');
+
+(async () => {
```

Comment:
I wonder whether we can trim the IIFE here as it adds extra cruft to the example. Especially since [top level await](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/await#top_level_await) is now a thing 👍